modules/core/src/copy.cpp

```diff
@@ -414,6 +414,46 @@ void Mat::copyTo( OutputArray _dst, InputArray _mask ) const
         copymask(ptrs[0], 0, ptrs[2], 0, ptrs[1], 0, sz, &esz);
 }
 
+// just 1 byte is valid
+#define CANT_APPLY_MEMSET int(0xDEADBEEF)
+
+static int check_apply_memset(const Mat *mat, const Scalar &s)
```

Comment:
```.diff
-const Mat *mat,
+const Mat& mat,
```

to make static analyzers happy (missing unnecessary zero pointer check)

Reply:
I agree.